src/infrastructure/figma/transformers/figma-file-transformer.ts

```diff
@@ -37,6 +39,7 @@ export const transformFileToAtlassianDesign = ({
 		status: AtlassianDesignStatus.NONE,
 		type: AtlassianDesignType.FILE,
 		lastUpdated: fileResponse.lastModified,
+		lastUpdatedBy: fileMetaResponse.file.last_touched_by,
```

Comment:
**issue:** Please, do a manual mapping here as well to include only necessary field and exclude unnecessary (e.g., as `email`).